charts/vector/values.yaml

```diff
@@ -145,8 +145,9 @@ command: []
 # args -- Override Vector's default arguments.
 args:
   - --config-dir
-  - "/etc/vector/"
-
+  - "/etc/vector/,/etc/local-config"
```

Comment:
what's the difference between /etc/vector/ and /etc/local-config ? @Gentleelephant @wanjunlei

Reply:
The directory /etc/vector/ stores the global configuration of vector, while /etc/local-config/ stores the configuration generated by secrets.

Reply:
> The directory /etc/vector/ stores the global configuration of vector, while /etc/local-config/ stores the configuration generated by secrets.

Then we can call it `custom-config` instead of `local-config`

Reply:
@Gentleelephant Please change like this.


```
        volumes:
        - name: data
          hostPath:
            path: /var/lib/vector/aggregator
            type: ''
        - name: global-config
          projected:
            sources:
              - configMap:
                  name: vector-aggregator
            defaultMode: 420
        - name: custom-config
          emptyDir: {}
        - name: host-time
          hostPath:
            path: /etc/localtime
            type: ''
      containers:
        - name: vector
          volumeMounts:
            - name: data
              mountPath: /vector-data-dir
            - name: global-config
              readOnly: true
              mountPath: /etc/vector/global
            - name: custom-config
              readOnly: true
              mountPath: /etc/vector/custom
            - name: host-time
              readOnly: true
              mountPath: /etc/localtime
```